client/state/jetpack-connect/selectors.js

```diff
@@ -10,4 +10,12 @@ const isCalypsoStartedConnection = function( state, siteSlug ) {
 	return false;
 };
 
-export default { isCalypsoStartedConnection };
+const getFlowType = function( state, site ) {
+	const siteSlug = site.slug.replace( /.*?:\/\//g, '' );
```

Comment:
the replace is probably just legacy... I think I recall that we were using the url before the slug, probably I changed it and I didn't realise that the filter wasn't required anymore